src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [visiblePeople, setVisiblePeople] = useState<Person[]>([]);
```

Comment:
Storing `visiblePeople` in a state will cause extra re-renders. Make it a variable